docs/guides/matter-repl.md

```diff
@@ -254,7 +254,7 @@ It provides some extra options, for example:
 optional arguments:
   -h, --help                show this help message and exit
   --app APP                 Path to local application to use, omit to use external apps.
-  --factoryreset            Remove app config and repl configs (/tmp/chip* and /tmp/repl*) before running the tests.
+  --factory-reset           Remove app config and repl configs (/tmp/chip* and /tmp/repl*) before running the tests.
```

Comment:
Not sure this was updated. I don't see the updated there...

Reply:
It was updated in #35886 but I've missed that help message at that time.